### PR TITLE
Add MineDestiny

### PR DESCRIPTION
**Pull request by Zsoz25K**
### General:
* [ ] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] No changes were made to the file's formatting.
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id.
* [ ] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string *(ex. `myserver`)*
  - [ ] `name`: a string *(ex. `MyServerPvP`)*
  - [ ] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [ ] `primaryAddress`: an address *(ex. `my.server`)*
  - [ ] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]`)*
  - [ ] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18`)*

### Logo Additions or Updates:
* [ ] My image is a `png` file.
* [ ] I uploaded my image to the `logos` directory.
* [ ] My image has a transparent background and is square (1:1 aspect ratio).
* [ ] My image is `512` pixels in width and height.
* [ ] My image's file name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1899,5 +1899,15 @@
         "primaryAddress": "play.sunrisecity.uk",
         "minecraftVersions": ["1.17", "1.18"],
         "primaryMinecraftVersion": "1.17"
+    },
+    {
+        "id": "minedestiny",
+        "name": "MineDestiny",
+        "addresses": [
+            "play.minedestiny.hu"
```
Comment (itsJordan) [inline servers.json:1907]:
```suggestion
            "minedestiny.hu"
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please review + implement my suggestion

**Comment by Zsoz25K** [comment]
The server only works with the play. address

File: servers.json (hunk 1 above)
Comment (itsJordan) [reply servers.json:1907]: Addresses within the 'addresses' array are not used for player connections.
They are used to match any possible subdomains to the same domain and therefor do not need to contain any subdomains.

**Comment by ThatKawaiiSam** [comment]
Hi @Zsoz25K,

We have moved to a new format for submitting servers and as such I will close this PR. Please submit a new PR in the new format.

**Pull request closed without merging**